pkg/cluster/internal/providers/docker/node.go

```diff
@@ -169,3 +154,51 @@ func (c *nodeCmd) SetStderr(w io.Writer) exec.Cmd {
 func (n *node) SerialLogs(w io.Writer) error {
 	return exec.Command("docker", "logs", n.name).SetStdout(w).SetStderr(w).Run()
 }
+
+func (c *nodeCmd) cpArgs() []string {
+	return []string {
+		"cp",
```

Comment:
Doesn't docker cp depend on a tar binary? That's probably *larger* than cp/mkdir and certainly more complex.

Reply:
I don't think so, I was able to create clusters using the haproxy image I built without cp/mkdir/kill in the image and had no issues copying the cfg into the container.  I believe docker is using tools from the host to tar/untar input and output.